hub/load-test-http/values.yaml

```diff
@@ -1,66 +1,53 @@
-### Defaults values and behaviors of various parameters that can be set
-### for this experiment chart.
+### Defaults values for the load-test-http experiment chart.
 
-### Lines where values can be set start with a single #
-### In contrast, description lines like this one start with ### 
+### The documentation follows Helm recommendations described in the URL below.
+### https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml
 
 ##################################
 
 ### url   HTTP(S) URL where the app receives GET or POST requests.
 ### This field is required.
-
-# url: https://example.com
+url: null
 
 ### headers   HTTP headers to be used in requests sent to the app.
 ### Specified as a map with key being header names, and values being header values.
+headers: null
 
-# headers:
-#   hello: "mars"
-#   goodbye: "pluto"
-
-### numQueries    Number of requests sent to the app. Default is 100
-
-# numQueries: 100
+### numQueries    Number of requests sent to the app.
+numQueries: 100
 
 ### duration    Duration for which requests are sent to the app.
 ### Value can be any Go duration string (https://pkg.go.dev/maze.io/x/duration#ParseDuration)
 ### This field is ignored if `numQueries` is specified.
+duration: 20s
```

Comment:
Per discussion in https://github.com/iter8-tools/iter8/issues/1164 `numQueries` cannot be defaulted to 100; this will override any duration the user sets.